opendbc/car/ford/values.py

```diff
@@ -124,7 +124,14 @@ class CAR(Platforms):
   FORD_ESCAPE_MK4 = FordPlatformConfig(
     [
       FordCarDocs("Ford Escape 2020-22", hybrid=True, plug_in_hybrid=True),
-      FordCarDocs("Ford Kuga 2020-22", "Adaptive Cruise Control with Lane Centering", hybrid=True, plug_in_hybrid=True),
+      FordCarDocs("Ford Kuga 2020-23", "Adaptive Cruise Control with Lane Centering", hybrid=True, plug_in_hybrid=True),
+    ],
+    CarSpecs(mass=1750, wheelbase=2.71, steerRatio=16.7),
+  )
+  FORD_ESCAPE_MK4_5 = FordCANFDPlatformConfig(
+    [
+      FordCarDocs("Ford Escape 2023-24", hybrid=True, plug_in_hybrid=True),
```

Comment:
There are two packages, `Co-Pilot360 Assist+ (68B)` and `Co-Pilot360 Assist 2.0 (68C)`, which both have Adaptive Cruise and Lane Centering.
2023 - https://www.mavericktruckclub.com/forum/attachments/2023-escape-order-guide-pdf.116038/
2024 - https://dealerimages.dealereprocess.com/image/upload/v1700240087/2024_Escape_obwtvs.pdf

The cars come with `Co-Pilot360 features` as standard but they only include Lane Keeping and AEB.

Reply:
I think it's fine to leave this as `Co-Pilot360 Assist+` like the Q3 Escape